site/_docs/contributing.md

```diff
@@ -27,9 +27,11 @@ following in mind:
   change is to review, the more likely it will be merged.
 * When submitting a pull request, please make judicious use of the pull request
   body. A description of what changes were made, the motivations behind the
-  changes and [any tasks completed or left to complete](http://git.io/gfm-tasks)
+  changes and [any tasks completed or left to complete][]
```

Comment:
I think it's fine for this to be > 80 characters. Our code's limit is more like 110 or 120 characters. Thanks!